plugins/pipelines/badge.js

```diff
@@ -3,20 +3,18 @@
 const joi = require('joi');
 const schema = require('screwdriver-data-schema');
 const workflowParser = require('screwdriver-workflow-parser');
-const tinytim = require('tinytim');
 const idSchema = schema.models.pipeline.base.extract('id');
+const { makeBadge } = require('badge-maker');
 
 /**
- * Generate Badge URL
- * @method getUrl
- * @param  {String} badgeService            Badge service url
+ * Generate Badge
+ * @method getBadge
  * @param  {Object} statusColor             Mapping for status and color
- * @param  {Function} encodeBadgeSubject    Function to encode subject
  * @param  {Array}  [buildsStatus=[]]       An array of builds
- * @param  {String} [subject='job']         Subject of the badge
+ * @param  {String} [label='pipeline']         Subject of the badge
  * @return {String}
  */
-function getUrl({ badgeService, statusColor, encodeBadgeSubject, buildsStatus = [], subject = 'pipeline' }) {
+function getBadge({ statusColor, buildsStatus = [], label = 'pipeline' }) {
```

Comment:
would it be better to move this function to a helper file like this: https://github.com/screwdriver-cd/screwdriver/blob/master/plugins/helper.js?